pointmatcher/DataPointsFilters/utils/octree.h

```diff
@@ -127,41 +118,44 @@ class Octree
 	DataContainer data;	
 	
 	std::size_t depth;
-	
+		
 public:
-	Octree();
-	Octree(const Octree<T>& o); //Deep-copy
-	Octree(Octree<T>&&o);
+	Octree_();
+	Octree_(const Octree_<T,dim>& o); //Deep-copy
+	Octree_(Octree_<T,dim>&&o);
 	
-	virtual ~Octree();
+	virtual ~Octree_();
 	
-	Octree<T>& operator=(const Octree<T>&o); //Deep-copy
-	Octree<T>& operator=(Octree<T>&& o);
+	Octree_<T,dim>& operator=(const Octree_<T,dim>&o);//Deep-copy
```

Comment:
A minor general remark: you are not very consistent with the spaces around operators and the `,`. The pointmatcher convention are if I'm right: single space before and after every binary, ternary operator symbols and single space after each `,`. Unary operators are different but I didn't observe issues. I'm not going to mark that everywhere. Also, sometimes there are lines purely consisting of white spaces. They should just be empty (special case of the "no trailing white spaces"-rule).